client/py/aou_workbench_client/auth.py

```diff
@@ -0,0 +1,82 @@
+"""OAuth against the AoU Workbench API using bearer tokens.
+
+The credentials are obtained by oauth2client, from one of:
+*   Google application default credentials (expected case, from notebook servers).
+*   A private key file, path specified in GOOGLE_APPLICATION_CREDENTIALS environment variable.
+
+For local development using a private key file:
```

Comment:
So... this is going to use service account credentials, which is different from the end user credentials we'd expect a user to have in a notebook.

It might be better to figure out the steps for getting creds file for the user set up here. Not sure what that looks like.

Reply:
(I guess technically, we expect the creds for a notebook to be that of their pet service account. It's just a little weird to use the server service account here.)

Reply:
Added a TODO. I agree it doesn't match how it'll work in notebooks; I think pet service accounts are still on their way, though (https://precisionmedicineinitiative.atlassian.net/browse/RW-32), so this is a reasonable stopgap for developing the client library.